### PR TITLE
Feat/hng 82 develop waitlist form to add user

### PR DESCRIPTION
**Pull request by toonami2907**
[

# Description

**Closes #issue_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x]This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

Screenshot_20-7-2024_115925_localhost

#Figma Link
https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=518-7790&m=dev

### CODE REVIEW

**Comment by psalm987** [comment]
Please review your screenshot. there's also no figma link attached

File: app/components/WaitListForm.tsx
Comment (SirhmVFX) [inline app/components/WaitListForm.tsx:1]: remove all your commentssss

**Review by SirhmVFX — CHANGES_REQUESTED** [review]

File: app/components/WaitListForm.tsx
Comment (toonami2907) [reply app/components/WaitListForm.tsx:1]: i have removed all comments from code

**Comment by toonami2907** [comment]
> Please review your screenshot. there's also no figma link attached

**Review by shalomtaiwo — CHANGES_REQUESTED** [review]
image

**Pull request closed without merging**